indico/modules/events/registration/client/js/form/fields/TimetableSessions.jsx

```diff
@@ -0,0 +1,296 @@
+    return acum || value.includes(id);
+  }, false);
+  const [isExpanded, setIsExpanded] = useState(display === 'expanded');
+
+  const isExpandedClass = isExpanded ? '' : 'hidden';
+
+  function handleHeaderClick() {
+    setIsExpanded(!isExpanded);
+  }
+
+  function handleHeaderChange() {
+    const newValue =
+      isChecked || isIndeterminate
+        ? value.filter(element => !childrenIds.includes(element))
+        : [...value, ...childrenIds];
+    onChange(newValue);
+  }
+
+  return (
+    <>
+      <Accordion styleName="equipment-accordion" className="ui-state-default">
+        <Accordion.Title active={isExpanded} index={0} onClick={handleHeaderClick}>
+          <Icon name="dropdown" />
+          <Checkbox
+            checked={isChecked}
+            onChange={handleHeaderChange}
+            indeterminate={isIndeterminate && !isChecked}
+            disabled={!allowFullDays}
+            label={label}
+          />
+        </Accordion.Title>
+      </Accordion>
+      {data
+        .map(item => camelizeKeys(item))
+        .map(({fullTitle, id}) => (
+          <dd className="grouped-fields" key={`session-block-checkbox-${id}`}>
+            <Checkbox
+              value={id}
+              onChange={handleValueChange}
+              checked={value.includes(id)}
+              label={fullTitle}
+              className={isExpandedClass}
+            />
+          </dd>
+        ))}
+    </>
+  );
+}
+
+SessionBlockHeader.propTypes = {
+  value: PropTypes.arrayOf(PropTypes.number).isRequired,
+  data: PropTypes.arrayOf(PropTypes.object).isRequired,
+  label: PropTypes.string.isRequired,
+  allowFullDays: PropTypes.bool.isRequired,
+  display: PropTypes.string.isRequired,
+  handleValueChange: PropTypes.func.isRequired,
+  onChange: PropTypes.func.isRequired,
+};
+
+export default function TimetableSessionsInput({
+  htmlId,
+  htmlName,
+  disabled,
+  isRequired,
+  allowFullDays,
+  display,
+  minimum,
+  maximum,
+}) {
+  const {eventId} = useSelector(getStaticData);
+  const {data: sessionData} = useIndicoAxios({url: timeTableURL({event_id: eventId})});
+
+  const minMsg = minimum > 0 ? `at least ${minimum}` : ``;
+  const and = minimum > 0 && maximum > 0 ? ` and ` : ``;
+  const maxMsg = maximum > 0 ? `no more than ${maximum}` : ``;
+
+  return (
+    <FinalField
+      id={htmlId}
+      name={htmlName}
+      component={TimetableSessionsComponent}
+      disabled={disabled}
+      required={isRequired}
+      allowFullDays={allowFullDays}
+      display={display}
+      minimum={minimum}
+      maximum={maximum}
+      sessionData={sessionData}
+      validate={value => {
+        if (value.length < minimum || value.length > maximum) {
+          return PluralTranslate.string(
+            'Please select {0}{1}{2} session block.',
+            'Please select {0}{1}{2} session blocks.',
+            maximum,
+            {0: minMsg, 1: and, 2: maxMsg}
+          );
```

Comment:
This makes it so that `at least ..`/`no more than ..` is not translatable since they are treated as parameters. The translators will just see `Please select {0}{1}{2} session block.` It's best not to concatenate translatable message i.e. use the full messages, even if you have to create more of them.